Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationDetailsSummary.cshtml

```diff
@@ -52,25 +52,27 @@
                             Organisation type
                         </dt>
                         <dd class="govuk-summary-list__value">
-                            @details?.OrganisationScheme
+                            @Constants.OrganisationSchemeType.OrganisationScheme[@details.OrganisationScheme!];
```

Comment:
Is there a way we could avoid using `!` please?

Reply:
I have added extension method